cluster-up/cluster/kind/common.sh

```diff
@@ -303,7 +303,7 @@ function down() {
     if [ -z "$($KIND get clusters | grep ${CLUSTER_NAME})" ]; then
         return
     fi
-    $KIND delete cluster --name=${CLUSTER_NAME}
-    docker rm -f $REGISTRY_NAME >> /dev/null
+    $KIND delete cluster --name=${CLUSTER_NAME} || true
+    docker rm -f $REGISTRY_NAME >> /dev/null || true
```

Comment:
do we need also `docker rm -f $REGISTRY_NAME >> /dev/null || true` ?
we didn't see it fail so far
i think we better remove the change here?